src/java.base/share/classes/java/lang/constant/DirectMethodHandleDescImpl.java

```diff
@@ -134,57 +134,34 @@ public MethodTypeDesc invocationType() {
 
     @Override
     public String lookupDescriptor() {
-        switch (kind) {
-            case VIRTUAL:
-            case SPECIAL:
-            case INTERFACE_VIRTUAL:
-            case INTERFACE_SPECIAL:
-                return invocationType.dropParameterTypes(0, 1).descriptorString();
-            case STATIC:
-            case INTERFACE_STATIC:
-                return invocationType.descriptorString();
-            case CONSTRUCTOR:
-                return invocationType.changeReturnType(CD_void).descriptorString();
-            case GETTER:
-            case STATIC_GETTER:
-                return invocationType.returnType().descriptorString();
-            case SETTER:
-                return invocationType.parameterType(1).descriptorString();
-            case STATIC_SETTER:
-                return invocationType.parameterType(0).descriptorString();
-            default:
-                throw new IllegalStateException(kind.toString());
-        }
+        return switch (kind) {
+            case VIRTUAL, SPECIAL,
```

Comment:
Nit: I prefer to have each case in a separate line (in this switch and also the switch in `resolveConstantDesc`.

Reply:
Hi Mandy. Thanks for your suggestion. I've now moved each case onto its own line. Please see  ccc9db7 / df5b34e